libs/custodian/provider/bitflyer.go

```diff
@@ -0,0 +1,319 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"sort"
+	"strings"
+
+	"github.com/brave-intl/bat-go/libs/clients"
+	"github.com/brave-intl/bat-go/libs/clients/bitflyer"
+	appctx "github.com/brave-intl/bat-go/libs/context"
+	errorutils "github.com/brave-intl/bat-go/libs/errors"
+	loggingutils "github.com/brave-intl/bat-go/libs/logging"
+	"github.com/gofrs/uuid"
+	"github.com/shopspring/decimal"
+)
+
+var (
+	// BitflyerTransferIDNamespace uuidv5 namespace for bitflyer transfer id creation
+	BitflyerTransferIDNamespace = uuid.Must(uuid.FromString("5b208c1d-e1c4-4799-bcc2-0b08b9c660f5"))
```

Comment:
Is this like an account ID for us? Sorry I'm not familiar with what the trasfterID is used for.